codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -275,16 +275,17 @@ private void generateOperationSerializerMiddleware(GenerationContext context, Op
             });
 
             writer.write("");
-
-            boolean withRestBindings = isOperationWithRestBindings(model, operation);
+            writer.write("request.Request.URL.Path = $S", httpTrait.getUri());
```

Comment:
When/how should the SDK's REST middleware treat PathRaw of URL?  This was one thing that the SDK had to add in order to ensure that HTTP client escaped the URL path exactly how the SDK signed it. Not sure if this can be done incrementally, or all at once when URL path parameters have been replaced.